tests/plugins/webp-uploads/load-tests.php

```diff
@@ -39,20 +42,29 @@ static function ( string $filename ) {
 
 	/**
 	 * Don't create the original mime type for JPEG images.
+	 *
+	 * @dataProvider data_provider_supported_image_types
+	 *
+	 * @test
```

Comment:
These `@test` annotations have been removed as of #1246

```suggestion
```

Reply:
oh, thanks. i didn't see that and noticed they were removed in a merge.